projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py

```diff
@@ -269,6 +269,7 @@ def deploy(
     logdir: Path,
     accounting_group_user: str,
     accounting_group: str,
+    state_flag: Optional[str] = None,
```

Comment:
I think this and the docstring description/position should be updated as well.

Reply:
Oh, and could you move the docstring description in the other function as well. Sorry, didn't notice that the argument had moved.